packages/react/src/primitives/View/View.tsx

```diff
@@ -27,7 +27,6 @@ const ViewPrimitive = <Element extends ElementType = 'div'>(
       'aria-label': ariaLabel,
       'data-testid': testId,
       disabled: isDisabled,
-      isDisabled: isDisabled,
```

Comment:
@jacoblogan Can you double check? - https://github.com/aws-amplify/amplify-ui/pull/1772/files#r865784417

Reply:
> this was added in because of the FieldGroupIconButton which is using the as property to render our primitive Button but the isDisabled flag was not being propagated down to the Button primitive.

I don't understand why this problem would show up after refactoring the CSS. Do you think this was a preexisting bug?